doordeck-sdk/src/commonMain/kotlin/com/doordeck/multiplatform/sdk/api/responses/ResponseError.kt

```diff
@@ -0,0 +1,9 @@
+package com.doordeck.multiplatform.sdk.api.responses
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+internal class ResponseError(
+    val code: Int? = null,
```

Comment:
Looking at `com.doordeck.core.exception.MappableException`, `code` could be a float style, e.g. `404.12`, admittedly I don't fractional error codes are in use anywhere but the intention was to allow specific error codes for any condition by using the decimal. I suggest adding a test to see what happens.

Reply:
Done!